common/pulp_docker/common/tarutils.py

```diff
@@ -114,3 +114,24 @@ def get_youngest_children(metadata):
                 pass
 
     return list(image_ids)
+
+
+def get_image_manifest(tarfile_path):
+    """
+    Given a path to a tarfile, which itself is the product of "skopeo copy"
+    this returns the decoded manifest.json file if it exists.
+
+    :param tarfile_path: full path to tarfile
+    :type tarfile_path:  basestring
+    :return:             decoded manifest json
+    :rtype               list or dict
+    """
+    image_manifest = []
```

Comment:
ok, now that makes sense. looks like they have a newer version which is 1.2 and it does contain indeed the manifest.json
https://github.com/moby/moby/blob/master/image/spec/v1.2.md